### PR TITLE
OSDOCS-9524: Added the ap-northeast-2 region

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
`enterprise-4.15+`

Issue:
[OSDOCS-9524](https://issues.redhat.com/browse/OSDOCS-9524)

Link to docs preview:
* [Regions and availability zones](http://file.rdu.redhat.com/eponvell/OSDOCS-9524_ap-northeast-2/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-sdpolicy-regions-az_rosa-service-definition) for ROSA Classic
* [Regions and availability zones](http://file.rdu.redhat.com/eponvell/OSDOCS-9524_ap-northeast-2/rosa_architecture/rosa_policy_service_definition/rosa-hcp-service-definition.html#rosa-sdpolicy-regions-az_rosa-hcp-service-definition) for ROSA with HCP
* [Comparing ROSA with hosted control planes and ROSA Classic](http://file.rdu.redhat.com/eponvell/OSDOCS-9524_ap-northeast-2/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html#rosa-hcp-classic-comparison_rosa-hcp-sts-creating-a-cluster-quickly)

QE review:
- [ ] QE has approved this change.

Additional information:
Added the `ap-northeast-2` region to ROSA with HCP docs.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.16` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.15. And any PR going into `main` must also target the latest version branch (enterprise-4.16).

If the update in your PR does NOT apply to version 4.16 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

File: modules/rosa-hcp-classic-comparison.adoc [hunk 1]
```diff
@@ -57,6 +57,8 @@
 * US East - N. Virginia (us-east-1)
 * US East - Ohio (us-east-2)
 * US West - Oregon (us-west-2)
```
Comment (dfitzmau) [inline modules/rosa-hcp-classic-comparison.adoc:59]: Hi @EricPonvelle .

In the ROSA service defintion document, the Regions and availability zones table references 

- ap-northeast-1 (Tokyo)
- ap-northeast-2 (Seoul) x 2

Should these entries belong in this file as HCP is the target medium.

Comment (EricPonvelle) [reply modules/rosa-hcp-classic-comparison.adoc:59]: There was a merge conflict. It should be corrected now.

Comment (dfitzmau) [reply modules/rosa-hcp-classic-comparison.adoc:59]: I still see teh entries in the provided preview link for [Regions and availability zones](http://file.rdu.redhat.com/eponvell/OSDOCS-9524_ap-northeast-2/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-sdpolicy-regions-az_rosa-service-definition) for ROSA Classic. Maybe the preview link needs a refresh on the PR (happens to me quiet a lot)?

File: modules/rosa-hcp-classic-comparison.adoc [hunk 2]
```diff
@@ -58,6 +58,8 @@
...
 * Africa - Cape Town (af-south-1)
+* Asia Pacific - Tokyo (ap-northeast-1)
```
Comment (dfitzmau) [inline modules/rosa-hcp-classic-comparison.adoc:61]: Consider sorting the list by region to make the list more scanable. All "northeast" are not grouped.

**Comment by dfitzmau** [comment]
/remove-label peer-review-in-progress

/label peer-review-done

LGTM, but with a presentation of regions comment suggested.

File: modules/rosa-hcp-classic-comparison.adoc (hunk 2 above)
Comment (EricPonvelle) [reply modules/rosa-hcp-classic-comparison.adoc:61]: That's intentional. I was matching AWS's presentation: https://docs.aws.amazon.com/general/latest/gr/rosa.html

File: modules/rosa-sdpolicy-am-regions-az.adoc [hunk 3]
```diff
@@ -68,11 +64,15 @@ endif::rosa-with-hcp[]
...
 * eu-west-3 (Paris)
 * eu-south-2 (Spain, AWS opt-in required)
+endif::rosa-with-hcp[]
 * eu-north-1 (Stockholm)
+ifndef::rosa-with-hcp[]
 * eu-central-2 (Zurich, AWS opt-in required)
 * me-south-1 (Bahrain, AWS opt-in required)
 * me-central-1 (UAE, AWS opt-in required)
 * sa-east-1 (São Paulo)
+* us-gov-east-1 - (AWS GovCloud - US-East)
```
Comment (mletalie) [inline modules/rosa-sdpolicy-am-regions-az.adoc:74]:
```suggestion
* us-gov-east-1 (AWS GovCloud - US-East)
```

File: modules/rosa-sdpolicy-am-regions-az.adoc [hunk 4]
```diff
@@ -68,11 +64,15 @@ endif::rosa-with-hcp[]
...
+* us-gov-west-1 - (AWS GovCloud - US-West)
```
Comment (mletalie) [inline modules/rosa-sdpolicy-am-regions-az.adoc:75]:
```suggestion
* us-gov-west-1 (AWS GovCloud - US-West)
```

**Comment by mletalie** [comment]
Suggestion to remove two hyphens, otherwise LGTM!

**Pull request merged**